cmake/Modules/EmscriptenHelpers.cmake

```diff
@@ -7,5 +7,5 @@ FUNCTION(mr_emscripten_pack_directory SRC DST)
 ENDFUNCTION()
 
 FUNCTION(mr_emscripten_set_async_func_list FUNC_LIST_FILE)
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s ASYNCIFY_IGNORE_INDIRECT -s ASYNCIFY_ADD=@${FUNC_LIST_FILE}" PARENT_SCOPE)
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s ASYNCIFY_IGNORE_INDIRECT -s  ASYNCIFY_ADD=@${FUNC_LIST_FILE}" PARENT_SCOPE)
```

Comment:
why extra space?

Reply:
added to trigger linking locally, can be reverted